platform-ajax/src/main/typescript/com/softicar/platform/ajax/engine/Rect.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * An immutable 2-dimensional rectangle.
+ */
+class Rect {
```

Comment:
Couldn't we just use `DOMRect` instead?

Reply:
nope, DOMRect has more fields that we want, I only want `x,y,width,height`

Reply:
I see